src/internal/SetBuildNumber/Directory.Packages.props.pp

```diff
@@ -45,18 +45,18 @@
     <PackageVersion Include="System.DirectoryServices" Version="4.7.0" />
     <PackageVersion Include="System.DirectoryServices.AccountManagement" Version="4.7.0" />
     <PackageVersion Include="System.IO.Compression" Version="4.3.0" />
-    <PackageVersion Include="System.IO.FileSystem.AccessControl" Version="4.6.0" />
+    <PackageVersion Include="System.IO.FileSystem.AccessControl" Version="4.7.0" />
     <PackageVersion Include="System.Reflection.Metadata" Version="1.6.0" />
     <PackageVersion Include="System.Security.Principal.Windows" Version="4.7.0" />
-    <PackageVersion Include="System.Text.Encoding.CodePages" Version="4.6.0" />
+    <PackageVersion Include="System.Text.Encoding.CodePages" Version="4.7.0" />
     <PackageVersion Include="System.Text.Json" Version="4.6.0" />
 
     <PackageVersion Include="Microsoft.AspNetCore.Owin" Version="3.1.13" />
     <PackageVersion Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.3" />
-    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp" Version="3.8.0" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp" Version="4.3.1" />
```

Comment:
Is this necessary? I vaguely remember needing to keep this lower to support VS2019, but that might not be correct.

Reply:
This change will require users of WixToolset.Dnc.HostGenerator to have VS2022 17.3 or later - https://github.com/dotnet/roslyn/blob/main/docs/wiki/NuGet-packages.md.

Reply:
Not strictly necessary. Is it necessary to keep it at `3.8.0` for VS2019 support?

Reply:
According to that page, it needs to be 3.x to support VS2019.

Reply:
It looks like VS2019 16.11 is the only supported version after this month so we should be able to go to `3.11.0`.

Reply:
Weird. Your second comment didn't show up until I refreshed the page. That's a good doc link.

Reply:
Updated to 3.11 and added the link to that documentation.

Reply:
Looks good.